mkdocs.yml

```diff
@@ -54,7 +54,7 @@ nav:
   - Meetings:
       - Overview: meetings.md
       - Community meeting (Sept'22): meetings/2022-09-amsterdam.md
-  - Project partners: partners.md
```

Comment:
note: I'm deliberately removing this menu item, as the "partners" page is very outdated currently...